index.js

```diff
@@ -0,0 +1,51 @@
+let books = JSON.parse(localStorage.getItem('books'));
+
+const handleStorage = (data) => {
+  localStorage.setItem('books', JSON.stringify(data));
+};
+
+const listOfBooks = document.querySelector('.list');
+const formBtn = document.querySelector('.form_button');
+
+function addBook(book) {
+  books.push({ id: Math.random(), ...book });
```

Comment:
[OPTIONAL] Using array methods that do not mutate the array like concat instead of push is the best practice.